projects/glitter-finance/index.js

```diff
@@ -21,7 +21,10 @@ async function solana_tvl() {
     const tvlResult = {};
 
     //Get sol balance from the SOL vault
-    const amount_sol = await getSolBalance(SOLANA_VAULT);
+    var amount_sol = await getSolBalance(SOLANA_VAULT);
+    if (!amount_sol || amount_sol == 'NaN') {
```

Comment:
@Elaixzar thanks for the PR, it looks good except for this check here, if some error occurs we would rather it fail than return 0 which will result in wrong tvl, can you remove it

Reply:
okay, I am merging it and fixing it.

Reply:
Looks like you already merged it, but noted for future reference.